typescript/cli/src/read/core.ts

```diff
@@ -23,9 +25,13 @@ export async function executeCoreRead({
     );
   }
 
+  if (!interchainAccountRouter) {
```

Comment:
for visibility, we discussed removing this and just rely on the chain addresses. in the future, we will have an ica command to update other icas